### DISCUSSION TITLE
Tethys as a Block Propagation Network (a.k.a: BloxRoute)?

### DISCUSSION THREAD

**Discussion by aldoborrero** [41north/tethys#12] [Ideas]
I was scrolling through my Twitter feed, when something caught my attention:

![image](https://user-images.githubusercontent.com/82811/187411468-01a92c3b-5a9d-4607-968a-c8a846a7970b.png)

[Navigating to the link](https://natsbyexample.com/examples/use-cases/cross-region-streams-supercluster/cli), we can see an example of such configuration. In this particular case, the cost is latency, but we can optimize for speed of delivery.

Opinions?

**Comment by brianmcgee** [comment]
Do you remember the client sonar buoy idea we talked about ages ago? That was my first though. This is a logical extension of that. If done right the system will naturally know which clients have which blocks and be able to source them regardless of location
